solutions/devsprint-matheus-reis-1/RealEstateApp/UI/Screens/PropertyDetails/Components/PropertyInfoCollectionViewCell.swift

```diff
@@ -0,0 +1,149 @@
+        configureSubviews()
+        configureSubviewsConstraints()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    public lazy var priceLabel: UILabel = {
+        let priceLabel = UILabel()
+        priceLabel.translatesAutoresizingMaskIntoConstraints = false
+        priceLabel.textAlignment = .left
+        priceLabel.font = .boldSystemFont(ofSize: 22)
+        priceLabel.textColor = .black
+        return priceLabel
+    }()
+    
+    public lazy var condominiumFeeLabel: UILabel = {
+        let condominiumFeeLabel = UILabel()
+        condominiumFeeLabel.translatesAutoresizingMaskIntoConstraints = false
+        condominiumFeeLabel.textAlignment = .left
+        condominiumFeeLabel.font = .boldSystemFont(ofSize: 14)
+        condominiumFeeLabel.textColor = .black
+        return condominiumFeeLabel
+    }()
+
+    private let urbanTaxLabel: UILabel = {
+        let urbanTaxLabel = UILabel()
+        urbanTaxLabel.translatesAutoresizingMaskIntoConstraints = false
+        urbanTaxLabel.textAlignment = .left
+        urbanTaxLabel.font = .boldSystemFont(ofSize: 14)
+        urbanTaxLabel.textColor = .black
+        return urbanTaxLabel
+    }()
+
+    public lazy var areaLabel: UILabel = {
+        let areaLabel = UILabel()
+        areaLabel.translatesAutoresizingMaskIntoConstraints = false
+        areaLabel.textAlignment = .left
+        areaLabel.font = .systemFont(ofSize: 14)
+        areaLabel.textColor = .gray
+        return areaLabel
+    }()
+
+    public lazy var bedroomsLabel: UILabel = {
+        let bedroomsLabel = UILabel()
+        bedroomsLabel.translatesAutoresizingMaskIntoConstraints = false
+        bedroomsLabel.textAlignment = .left
+        bedroomsLabel.font = .systemFont(ofSize: 14)
+        bedroomsLabel.textColor = .gray
+        return bedroomsLabel
+    }()
+
+    public lazy var bathroomsLabel: UILabel = {
+        let bathroomsLabel = UILabel()
+        bathroomsLabel.translatesAutoresizingMaskIntoConstraints = false
+        bathroomsLabel.textAlignment = .left
+        bathroomsLabel.font = .systemFont(ofSize: 14)
+        bathroomsLabel.textColor = .gray
+        return bathroomsLabel
+    }()
+
+    public lazy var garageLabel: UILabel = {
+        let garageLabel = UILabel()
+        garageLabel.translatesAutoresizingMaskIntoConstraints = false
+        garageLabel.textAlignment = .left
+        garageLabel.font = .systemFont(ofSize: 14)
+        garageLabel.textColor = .gray
+        return garageLabel
+    }()
+
+    public lazy var addressLabel: UILabel = {
+        let addressLabel = UILabel()
+        addressLabel.translatesAutoresizingMaskIntoConstraints = false
+        addressLabel.textAlignment = .left
+        addressLabel.font = .systemFont(ofSize: 14)
+        addressLabel.textColor = .gray
+        return addressLabel
+    }()
+
+func configureSubviews() {
+    addSubview(priceLabel)
```

Comment:
Lys, se você achar que é proveitoso, como todos os elementos estão dentro de um mesmo elemento ( uma view, nesse caso ), você poderia adicionar todos de uma vez , dentro de um array.
```suggestion
[priceLabel,condominiumFeeLabel,urbanTaxLabel,areaLabel,bedroomsLabel,bathroomsLabel,garageLabel,addressLabel].forEach(addSubview)
```